### PR TITLE
Change default aspect for past tense

### PR DESCRIPTION
**Pull request by nerixim**
fixes https://github.com/rossmeissl/verbs/issues/15

@rossmeissl maybe make a (minor) release before merging this?

### CODE REVIEW

File: lib/verbs/conjugator.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-# * :tense => :past or :present or :future
-# * :person => :first or :second or :third
-# * :plurality => :singular or :plural
-# * :aspect => :habitual or :perfect or :perfective or :progressive or :prospective
-# * :mood => :indicative or :imperative or :subjunctive
-# Respective defaults are :present, :third, :singular, :habitual, and :indicative
+# * :tense => :past or :present or :future (default: :present)
+# * :person => :first or :second or :third (default: :third)
+# * :plurality => :singular or :plural (default: :singular)
+# * :aspect => :habitual or :perfect or :perfective or :progressive or :prospective (default: :habitual)
```
Comment (rossmeissl) [inline lib/verbs/conjugator.rb:5]: Here don't you have to indicate that default is perfective when tense is past and habitual otherwise?

**Review by rossmeissl — APPROVED** [review]
I've made a pre-merge release for this (2.3.1) and will do a major bump when this is merged.

File: lib/verbs/conjugator.rb (hunk 1 above)
Comment (nerixim) [reply lib/verbs/conjugator.rb]: missed this 🙏

**Comment by rossmeissl** [comment]
3.0.0 has been released. Thanks again @nerixim !

**Pull request merged**